packages/manager/apps/telecom-task/index.js

```diff
@@ -0,0 +1,7 @@
+import 'script-loader!jquery'; // eslint-disable-line
+import 'script-loader!lodash'; // eslint-disable-line
+
+import angular from 'angular';
+import '@ovh-ux/manager-telecom-task';
```

Comment:
```suggestion
import ovhManagerTelecomTask from '@ovh-ux/manager-telecom-task';
```

and then inject the module by using `ovhManagerTelecomTask` variable name.